nuc_morph_analysis/analyses/density/visualize_filter_for_uncaught_pseudo_cell_artifacts.py

```diff
@@ -1,4 +1,7 @@
 #%%
+# this code visualizes cells identified in this workflow `remove_uncaught_pseudo_cell_artifacts(df, apply_to_nucleus_too, verbose)`
```

Comment:
Can we update the remove_uncaught_pseudo_cell_artifacts to just filter all densities below 0.2 rather than just the ones within colony depth of 3 of the edge? It would make writing the methods section a lot simpler. Also if that is an outragiously large psuedo cell are the ones in the center with that value valid measurements?? Based on the density plot it looks like Medium is the only one that has nuclei that dip thaaaatttt low. But it is truely at the lower bound. LMK